mappings/net/minecraft/scoreboard/ScoreboardState.mapping

```diff
@@ -15,5 +15,5 @@ CLASS net/minecraft/class_273 net/minecraft/scoreboard/ScoreboardState
 		ARG 1 tag
 	METHOD method_1222 displaySlotsToTag (Lnet/minecraft/class_2487;)V
 		ARG 1 tag
-	METHOD method_32481 fromTag (Lnet/minecraft/class_2487;)Lnet/minecraft/class_273;
+	METHOD method_32481 fromNbt (Lnet/minecraft/class_2487;)Lnet/minecraft/class_273;
```

Comment:
Those `*From/Tag` methods also need to be renamed to use `read*FromNbt`. (and `displaySlotsToTag` to `writeDisplaySlotsToNbt`)

Reply:
Should be good now.